app/controllers/road-sign-concepts/new.js

```diff
@@ -6,7 +6,8 @@ import { inject as service } from '@ember/service';
 export default class RoadsignConceptsNewController extends Controller {
   @tracked isSaving = false;
   @service router;
-
+  @service fileService;
+  file;
```

Comment:
Not in reset controller but there is indeed something that has to change. when opting for an url instead of a file, file must be set to null. resolved in upcomming commit

Reply:
I think the route also needs to reset this? Otherwise if you select the file and leave the route, it will still be in the controller state.